liqd_product/config/settings/base.py

```diff
@@ -298,5 +303,10 @@
 A4_MAP_ATTRIBUTION = '&copy; <a href="http://openstreetmap.org/copyright">OpenStreetMap</a> contributors'
 A4_MAP_BOUNDING_BOX = ([[52.3517, 13.8229], [52.6839, 12.9543]])
 
+A4_DASHBOARD = {
+    'PROJECT_DASHBOARD_CLASS': 'meinberlin.apps.dashboard2.ProjectDashboard',
```

Comment:
should point to liqd_product...

Reply:
I thought we could use the default core dashboard until needed otherwise.

Reply:
the TypedDashboard is required to create/edit external projects

Reply:
I was not sure that external projects are required in the product.

Reply:
okay, thats a good point :) then we'd be able to drop the whole custom dashboard logic. but we should remove external projects from base.py then